src/containerapp/setup.py

```diff
@@ -17,7 +17,7 @@
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
 
-VERSION = '0.3.20'
+VERSION = '0.3.22'
```

Comment:
It seems that version `0.3.21` has not been released yet? Can this PR be released together in version `0.3.21`?